File: 250225/재귀함수를 이용한 3N + 1 수열/3n-plus-1-sequence-with-recursive-function.py
def count(n):
    if (n == 1):
        return 0;
    if (n % 2 == 0):
        return count(n // 2) + 1;
    return count(3 * n + 1) + 1;

n = int(input())

print(count(n))

File: 250225/재귀함수를 이용한 3N + 1 수열/3n-plus-1-sequence-with-recursive-function.cpp
#include <iostream>

using namespace std;

int n;

int count(int n) {
    if (n == 1)
        return 0;
    if (n % 2 == 0)
        return count(n / 2) + 1;
    return count(3 * n + 1) + 1;
}

int main() {
    cin >> n;

    cout << count(n);
    return 0;
}